2_process.R

```diff
@@ -12,8 +12,9 @@ p2_targets_list <- list(
   
   # Subset harmonized WQP data to lower DRB
   tar_target(
-    p2_filtered_wqp_data_subset,
-    subset_wqp_sites(p1_lowerdrb_boundary,p2_filtered_wqp_data)),
+    p2_wqp_data_subset_csv,
```

Comment:
What do you think about making this an object targets instead of file targets? I think it's quite a bit more convenient to do that. Then we don't have to deal with the file io. At first I was thinking that I'd be accessing these via Python, but I'm finding it more convenient to stay in R until even further down the road. We could probably take out the `p1_daily_data_csv` and `p1_inst_data_csv` targets too.

Reply:
> We could probably take out the p1_daily_data_csv and p1_inst_data_csv targets too.

Right now, `p1_daily_data_csv` and `p1_inst_data_csv` are called within `3_visualize/src/overview_plots.py` so if I deleted those targets the pipeline wouldn't build. I could try to emulate some of the figures in R if you'd prefer to get rid of these file targets (probably in a new issue).

Reply:
Ha yeah. I actually just wrote this below :)
https://github.com/USGS-R/drb-do-ml/pull/32#issuecomment-998305511

>I'm actually kind of thinking we maybe should take out the Python plots too 😄. They kind of take a while to build and then without them we can get rid of those csv targets in 1_fetch.R. They were nice to have to get a feel for the data, but I'm not sure we need to keep them around. I could be pursuaded otherwise tho about both the python targets and the plots from r.

>Maybe we do a happy medium and produce one summary plot with ggplot (to avoid the python rigmarole)

Reply:
So yeah. My vote would be for "one summary plot with ggplot". And I agree that this would be better in a separate issue/PR.

Reply:
OK, I am going to leave `p1_daily_data_csv` and `p1_inst_data_csv` for now (and `overview_plots.py`). I've opened a new issue (#33) for revisiting these summary plots using R (and the object targets as opposed to file targets).